docs/pages/management/admin/trustedclusters.mdx

```diff
@@ -1,500 +1,601 @@
+consists of the Teleport Auth Service, the Teleport Proxy Service, and the 
+Teleport services that manage access to resources in your infrastructure.
+With Teleport, you can also partition your infrastructure into multiple connected 
+clusters to allow the users of one cluster—the **root cluster**—to connect to resources 
+in another cluster—the **leaf cluster**—while authenticated using a single 
+Teleport Auth Service. 
 
-Trusted Clusters allow the users of one cluster, the **root cluster**, to
-seamlessly connect to resources in another cluster, the **leaf cluster**, while
-remaining authenticated with only a single Auth Service. The leaf cluster can
-run behind a firewall without any ingress ports open.
+After you establish a trust relationship between the root and leaf clusters, the leaf cluster can
+run behind a firewall without any inbound ports open. The leaf cluster creates an outbound reverse 
+SSH tunnel to the root cluster and keeps the tunnel open.
+When a user tries to connect to any resource in the leaf cluster, the connection goes through the 
+reverse tunnel using the root cluster's proxy.
 
-Uses for Trusted Clusters include:
+## Who uses trusted clusters?
+
+Most organizations don't need to configure trusted clusters. In most cases, you can add 
+reverse-tunnel proxies and enroll and manage thousands of resources without creating a new 
+cluster.
+
+However, there are a few specific scenarios where trusted clusters can be particularly useful.
+For example, if you have a large and widely-distributed infrastructure or must provide access
+to resources for external agencies, contractors, or clients, you might benefit from setting up 
+a trusted cluster. The most common use cases for trusted clusters include:
 
 - Managed service providers (MSP) remotely managing the infrastructure of their clients.
 - Device manufacturers remotely maintaining computing appliances deployed on premises.
 - Large cloud software vendors managing multiple data centers using a common proxy.
 
-Here is an example of an MSP using Trusted Clusters to obtain access to client clusters:
-![MSP Example](../../../img/trusted-clusters/trusted-clusters@1.5x.svg)
-
-A leaf cluster always creates an outbound reverse SSH tunnel to the root cluster and keeps the tunnel open.
-
-<Notice type="tip">
-Individual nodes and proxies can create reverse tunnels to proxy services without creating a new cluster.
-You don't need to set up a trusted cluster just to connect a couple of servers, kubernetes clusters or
-databases behind a firewall. For more information, see how to [join a service to your cluster](../../agents/join-services-to-your-cluster.mdx).
-</Notice>
-
-When a user tries to connect to any resource inside the leaf cluster using the root's proxy
-the connection goes through the reverse tunnel.
-
-This guide will explain how to:
-
-- Add and remove Trusted Clusters using CLI commands.
-- Enable/disable trust between clusters.
+## How it works
```

Comment:
This section looks a lot like our architecture page. Would it make sense to remove it?

Another option would organize all of our Trusted Cluster docs into a dedicated subsection (as with some of our other features, like Device Trust, Login Rules, etc.) so we can reduce repetition between the how-to guide and the architecture guide.

Reply:
Most of the architecture information was already in both pages with minimal differences. My intent was to remove the architecture page and cover everything in one place, either on a page or a group of pages but not duplicated.